src/snowflake/sqlalchemy/custom_commands.py

```diff
@@ -42,24 +42,25 @@ def __init__(self, command):
         def __repr__(self):
             if self.command == "INSERT":
                 sets, sets_tos = zip(*self.set.items())
-                return "WHEN NOT MATCHED{} THEN {} ({}) VALUES ({})".format(
-                    " AND %s" % self.predicate if self.predicate is not None else "",
-                    self.command,
-                    ", ".join(sets),
-                    ", ".join(map(str, sets_tos)),
-                )
+                return f"""
+WHEN NOT MATCHED\
+{f" AND {str(self.predicate if self.predicate is not None else '')}"} \
+THEN {self.command} ({', '.join(sets)}) \
+VALUES ({', '.join(map(str, sets_tos))})
+"""
             else:
                 # WHEN MATCHED clause
                 sets = (
                     ", ".join([f"{set[0]} = {set[1]}" for set in self.set.items()])
                     if self.set
                     else ""
                 )
-                return "WHEN MATCHED{} THEN {}{}".format(
-                    " AND %s" % self.predicate if self.predicate is not None else "",
-                    self.command,
-                    " SET %s" % sets if self.set else "",
-                )
+                return f"""\
+WHEN MATCHED\
+{f" AND {str(self.predicate if self.predicate is not None else '')}"} \
```

Comment:
Same for `str` here

Reply:
same comment as above, `str()` cast for equivalent "%s"